web/themes/custom/nysenate_theme/src/templates/layout/page.html.twig

```diff
@@ -107,6 +107,8 @@
 
     {{ page.help }}
 
+    {{ page.above_content }}
```

Comment:
@routinet content in the new region gets rendered through this simple addition to the main page template.

Reply:
What I mean to ask is how `page.above_content` gets its content, but I see now it is the YML specifying which blocks are part of the new region.